src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java

```diff
@@ -850,6 +852,31 @@ public void bootstrap() throws Exception {
                         git.commit().setMessage(INITIAL_COMMIT).call();
                     }
 
+                    git.close();
+                } catch (GitAPIException err) {
+                    logger.error("error creating initial commit for global configuration", err);
+                }
+            } else {
+                // rsync blueprints
+                String bootstrapFolderPath = this.ctx.getRealPath(FILE_SEPARATOR + BOOTSTRAP_REPO_PATH + FILE_SEPARATOR + BOOTSTRAP_REPO_GLOBAL_PATH + FILE_SEPARATOR + studioConfiguration.getProperty(BLUE_PRINTS_PATH));
+                Path source = java.nio.file.FileSystems.getDefault().getPath(bootstrapFolderPath);
+                Path globalConfigPath = helper.buildRepoPath(GitRepositories.GLOBAL);
+                Path blueprintsPath = Paths.get(globalConfigPath.toAbsolutePath().toString(), studioConfiguration.getProperty(BLUE_PRINTS_PATH));
+                String[] cmd = new String[] {"rsync", "-avz", "--delete", source.toAbsolutePath().toString() + FILE_SEPARATOR, blueprintsPath.toAbsolutePath().toString() };
```

Comment:
Will this work on Windows?

Reply:
How about we delete and recreate the blueprints folder if something has changed?

Reply:
Perhaps a checksum file or a date file and we check/decide.

Reply:
checksum file or timestamp file is a nice idea, the question is how do we automate it (if we rely on the user to update it, we might get into a problem if he forgets)

Reply:
the idea is to create a signature-file based on the war file's build number when starting up, we look at that file and if it's different from our build number, we delete the folder for the BPs and create them again and update the file